__tests__/utils/env-helpers.test.ts

```
@@ -0,0 +1,45 @@
+
+describe('getAppUrl', () => {
+  it('returns window location origin on client side', () => {
+    // @ts-ignore
+    global.window = { location: { origin: 'https://client.example.com' } }
+    const url = getAppUrl()
+    expect(url).toBe('https://client.example.com')
+  })
+
+  it('uses NEXT_PUBLIC_SITE_URL on server side', () => {
+    // @ts-ignore
+    delete global.window
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)